code/datums/supply_packs/food.dm

```diff
@@ -55,11 +55,140 @@
+					/obj/item/reagent_container/food/snacks/grown/eggplant,
+					/obj/item/reagent_container/food/snacks/grown/eggplant,
+					/obj/item/reagent_container/food/snacks/grown/eggplant,
+					/obj/item/reagent_container/food/snacks/grown/chili,
+					/obj/item/reagent_container/food/snacks/grown/chili,
+					/obj/item/reagent_container/food/snacks/grown/chili,
+					/obj/item/reagent_container/food/snacks/grown/chili,
+					/obj/item/reagent_container/food/snacks/grown/pumpkin,
+					/obj/item/reagent_container/food/snacks/grown/pumpkin,
+					/obj/item/reagent_container/food/snacks/grown/pumpkin,
+					/obj/item/reagent_container/food/snacks/grown/pumpkin,
+					/obj/item/reagent_container/food/snacks/grown/cabbage,
+					/obj/item/reagent_container/food/snacks/grown/cabbage,
+					/obj/item/reagent_container/food/snacks/grown/cabbage,
+					/obj/item/reagent_container/food/snacks/grown/cabbage,
+					)
+	cost = 7
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "\improper Vegetable crate"
+	group = "Food"
+
+/datum/supply_packs/fruits
+	name = "banana,apple,lemon,lime,orange,berries,cherries(x4)"
+	contains = list(
+					/obj/item/reagent_container/food/snacks/grown/banana,
+					/obj/item/reagent_container/food/snacks/grown/banana,
+					/obj/item/reagent_container/food/snacks/grown/banana,
+					/obj/item/reagent_container/food/snacks/grown/banana,
+					/obj/item/reagent_container/food/snacks/grown/apple,
+					/obj/item/reagent_container/food/snacks/grown/apple,
+					/obj/item/reagent_container/food/snacks/grown/apple,
+					/obj/item/reagent_container/food/snacks/grown/apple,
+					/obj/item/reagent_container/food/snacks/grown/lemon,
+					/obj/item/reagent_container/food/snacks/grown/lemon,
+					/obj/item/reagent_container/food/snacks/grown/lemon,
+					/obj/item/reagent_container/food/snacks/grown/lemon,
+					/obj/item/reagent_container/food/snacks/grown/orange,
+					/obj/item/reagent_container/food/snacks/grown/orange,
+					/obj/item/reagent_container/food/snacks/grown/orange,
+					/obj/item/reagent_container/food/snacks/grown/orange,
+					/obj/item/reagent_container/food/snacks/grown/lime,
+					/obj/item/reagent_container/food/snacks/grown/lime,
+					/obj/item/reagent_container/food/snacks/grown/lime,
+					/obj/item/reagent_container/food/snacks/grown/lime,
+					/obj/item/reagent_container/food/snacks/grown/berries,
+					/obj/item/reagent_container/food/snacks/grown/berries,
+					/obj/item/reagent_container/food/snacks/grown/berries,
+					/obj/item/reagent_container/food/snacks/grown/berries,
+					/obj/item/reagent_container/food/snacks/grown/cherries,
+					/obj/item/reagent_container/food/snacks/grown/cherries,
+					/obj/item/reagent_container/food/snacks/grown/cherries,
+					/obj/item/reagent_container/food/snacks/grown/cherries,
+					)
+	cost = 7
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "\improper fruits crate"
+	group = "Food"
```

Comment:
you could avoid writing cost, containertype, and group if you used inheritance

Reply:
i made an parent object called plants